projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration.ts

```diff
@@ -44,6 +44,24 @@ export function goToConfigurationPage(
   });
 }
 
+/**
+ * Navigates to the CPQ product configuration page.
+ *
+ * @param {string} shopName - shop name
+ * @param {string} productId - Product ID
+ * @return {Chainable<Window>} - New configuration window
+ */
+export function goToCPQConfigurationPage(
```

Comment:
We try to avoid code duplication.
Please, add an additional parameter that defines the configurator type in the url:
**const location = `/${shopName}/en/USD/configure/${cpq}/product/entityKey/${productId}`;**